bot/seasons/evergreen/issues.py

```diff
@@ -4,54 +4,71 @@
 from discord.ext import commands
 
 from bot.constants import Colours
+from bot.constants import Emojis
 from bot.decorators import override_in_channel
 
 log = logging.getLogger(__name__)
 
+icons = {"issue": Emojis.issue,
```

Comment:
For constants, PEP8 [specifies](https://www.python.org/dev/peps/pep-0008/#constants) all-capitals with underscores separating words.

Reply:
The field names were fine, `icon` should be `ICON`